components/toast/Toast.js

```diff
@@ -48,6 +48,36 @@ class Toast extends PureComponent {
     }, timeout);
   };
 
+  renderCustomAction = (action, actionLabel) => {
+    if (action) {
+      return <LinkButton className={theme['action-link']} inverse label={actionLabel} onClick={action} />;
+    }
+    return false;
```

Comment:
you can write this:
```js
renderCustomAction = (action, actionLabel) => action && <LinkButton className={theme['action-link']} inverse label={actionLabel} onClick={action} />;
```
If action is falsy the double ampersand will return false, if action is truthy it will return the component.

Reply:
You can actually do that for every single one, returning false will render nothing.